### DISCUSSION TITLE
Extend system requirements

### DISCUSSION THREAD

**Discussion by Drovosek01** [milanvarady/Applite#15] [Q&A] [answered]
Thank you for a good free app, but what is the reason to use only the latest stable version of macOS to launch applications?
Many users do not use the latest generation of macOS.

Please make this app available at least on macOS Catalina 10.15

**Comment by milanvarady** [comment, marked as answer]
I wanted to at least target Big Sur, but the problem is that Apple doesn't understand the term "backwards compatibility" when it comes to SwiftUI. Every new feature they add to SwiftUI is only for the latest version of macOS. And since SwiftUI is a relatively new UI framework, they add things that are missing or greatly improved from the previous solution.

In conclusion, if I wanted to add extended support, I would have to put a ton of work into workarounds for features that are Ventura only.

**Reply by milanvarady** [reply]
It is possible that support could be extended to Monterey in the future, as this would only require a manageable amount of work.

**Comment by 4bitfocus** [comment]
Putting in my vote... I would love to see at least Monterey support. A lot of the hackintosh crowd will be on Monterey, but probably not able to move past that for some time. I can see a lot of overlap from the hackintosh crowd and the home brew crowd.
